cli/services/kibana.go

```diff
@@ -0,0 +1,239 @@
+package services
```

Comment:
This file is in the `cli` side so that the unit tests are run in CI with no change.